src/commands/ext-update.ts

```diff
@@ -229,10 +190,8 @@ export default new Command("ext:update <extensionInstanceId> [updateSource]")
           return;
         }
       }
-      const isOfficial =
-        newSourceOrigin === SourceOrigin.OFFICIAL_EXTENSION ||
-        newSourceOrigin === SourceOrigin.OFFICIAL_EXTENSION_VERSION;
-      await displayChanges(existingSpec, newSpec, isOfficial);
+
+      await displayChanges(existingSpec, newSpec, false);
```

Comment:
I remember this logic was supposed to display before/after sources only if the extension wasn't official, I believe. Worth checking with Sonakshi to see if it's okay to deprecate this logic.

If it is okay to deprecate that logic, then I think we can remove the last parameter altogether from displayChanges()

Reply:
Cleaned this up as well, good eye!